tests/components/uvc/test_camera.py

```diff
@@ -267,7 +309,8 @@ def test_camera_image_tries_login_bails_on_failure(self):
     def test_camera_image_logged_in(self):
         """Test the login state."""
         self.uvc._camera = mock.MagicMock()
-        assert self.uvc._camera.get_snapshot.return_value == self.uvc.camera_image()
+        assert self.uvc._camera.get_snapshot.return_value == self.uvc.camera_image(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4643140)